pandas/core/resample.py

```diff
@@ -1106,16 +1106,16 @@ def f(  # type: ignore[misc]
     setattr(Resampler, name, f)
 
 
-for method in ["sum", "prod", "min", "max", "first", "last"]:
-    _add_downsample_kernel(method, ("numeric_only", "min_count"))
-for method in ["median"]:
-    _add_downsample_kernel(method, ("numeric_only",))
-for method in ["sem"]:
-    _add_downsample_kernel(method, ("ddof", "numeric_only"))
-for method in ["ohlc"]:
-    _add_downsample_kernel(method, ())
-for method in ["nunique"]:
-    _add_downsample_kernel(method, (), SeriesGroupBy)
+for mthd in ["sum", "prod", "min", "max", "first", "last"]:
```

Comment:
would `_method` work? as it's just used in the loop

Reply:
Yeah, that makes sense. Changed `mthd` to `_method`